de.fhg.igd.equinox.test.app/src/de/fhg/igd/equinox/test/app/runner/util/GlobUtil.java

```diff
@@ -0,0 +1,110 @@
+package de.fhg.igd.equinox.test.app.runner.util;
+
```

Comment:
Should the `Copyright (c) 2022 wetransform GmbH`text be added?

Reply:
I would not in that case as this class' content basically solely comes from the referenced StackOverflow link (where there is a comment that the author grants it to public domain).